src/modules/UI/scenes/SendConfirmation/reducer.js

```diff
@@ -17,42 +18,90 @@ export const sendConfirmation = (state: SendConfirmationState = initialState, ac
         forceUpdateGuiCounter++
       }
       if (!parsedUri) return { ...state, forceUpdateGuiCounter, error, transaction }
+
```

Comment:
are you sure that you that the object `state.parsedUri` is not undefined here?
if not, you might want to init it just in case so that lines like:
!isEqual(state.parsedUri.metadata, metadata) 
won't crash the app

Reply:
im not sure. but the reducer and actions make it difficult to tell
i dont think i changed anything except in the code that would affect this